webroot/js/message.js

```diff
@@ -261,4 +265,14 @@ class MessagingInterface {
 			jumpToBottom(this.scrollableMessagesContainer);
 		}
 	}
+
+	async getChatHistory() {
```

Comment:
I would actually keep this function in app.js, where it handles all the other data fetching for the app.
I was hoping to just keep this object, mainly for enabling functionality in the DOM.